lib/cli-engine.js

```diff
@@ -193,24 +198,40 @@ function processText(text, configHelper, filename) {
     }
 
     if (processor) {
+        debug("Using processor");
         var parsedBlocks = processor.preprocess(text, filename);
         var unprocessedMessages = [];
         parsedBlocks.forEach(function(block) {
             unprocessedMessages.push(eslint.verify(block, config, filename));
         });
         messages = processor.postprocess(unprocessedMessages, filename);
+
+        // TODO(nzakas): Figure out how fixes might work for processors
```

Comment:
Should theoretically work the same way as for regular code, it should be applied before `postprocess`